source/threads.tex

```diff
@@ -11316,9 +11316,11 @@
 \begin{itemize}
 \item
 \tcode{F::operator()} is a non-static member function and
-\tcode{decltype(\brk{}\&F::operator())} is of the form
+\tcode{decltype(\brk{}\&F::operator())} is either of the form
 \tcode{R(G::*)(A...)}~\cv{}~\tcode{\opt{\&}~\opt{noexcept}}
-for a class type \tcode{G}, or
+or of the form
+\tcode{R(*)(G, A...)~\opt{noexcept}}
+for a type \tcode{G}, or
```

Comment:
@brevzin, @jwakely, @CaseyCarter et al: I integrated the incoming wording with the prior change of CWG-12/["static operator()"](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2022/p1169r4.html). Could you please have a look if the merged wording is acceptable?

Reply:
@timsong-cpp also.

Reply:
LGTM

Reply:
CWG-12 assumes this was applied, so the end result should be exactly what's shown in CWG-12, and it looks like that is the case.

Reply:
Thank you! Oh yes indeed, there's even an note to the editor in LWG3617 to this effect, perfect!

Reply:
Oh, except that the CWG-17 paper says `G cv-ref_opt` where LWG3617 just says `G`.

Reply:
But I suppose since `G` is just "a type", that can include qualifiers.